index.js

```diff
@@ -17,7 +17,7 @@ const slackClient = new SlackClient(slackEventsAdapter, slackWebClient);
 
 registerEvents(slackClient);
 
-const port = process.env.APP_PORT || 3000;
+const port = process.env.PORT || 3000;
```

Comment:
EB automatically sets PORT to the port to which the proxy server passes traffic.

Reply:
Can we set a flag here for production w/ ebs?